src/CanvasChart.js

```diff
@@ -205,38 +209,40 @@ export default function CanvasChart({
         : task.length * monthWidth - MARGIN;
       const taskHeight = TASK_BAR_HEIGHT;
 
-      drawRoundedRect(
-        context,
-        taskX,
-        taskY,
-        taskWidth,
-        taskHeight,
-        CORNER_RADIUS
-      );
-      context.fillStyle = task.isOngoing ? "rgba(0,0,0,.05)" : color;
-      context.fill();
-
-      if (task.isOngoing) {
-        const chunkWidth =
-          (task.length * monthWidth) / (MONTHS.length - task.month);
+      if (!task.isMilestone) {
```

Comment:
This type breaks the dependency line. (To be fair, other cases were broken too– fixed with 6d6fae2).

But for example:
https://planner-git-fork-peterdresslar-main-bvaughn.vercel.app/#encoded=eyJ0YXNrcyI6W3siaWQiOiJtaWxlIiwibmFtZSI6IkV4YW1wbGUgbWlsZXN0b25lIiwibW9udGgiOjAsImlzTWlsZXN0b25lIjp0cnVlfSx7Im1vbnRoIjoxLCJsZW5ndGgiOjEsIm5hbWUiOiJ0ZXN0Iiwib3duZXIiOiJ0ZWFtIiwiZGVwZW5kZW5jeSI6Im1pbGUifV0sIm93bmVycyI6e319

Seems like ideally, the dependency line should align with the middle of the milestone diamond (horizontally).

Reply:
This should be fixed, in a rather ternary way, in #a986476. There are still some possible jank configurations with all dependencies, and I didn't want to do anything too refactor-y because of that.